mozilla-release/toolkit/locales/en-US/chrome/mozapps/downloads/downloads.properties

```diff
@@ -32,14 +32,14 @@ offlineCancelDownloadsAlertTitle=Cancel All Downloads?
 offlineCancelDownloadsAlertMsg=If you go offline now, 1 download will be canceled. Are you sure you want to go offline?
```

Comment:
It's a bit suspicious that this file has no changes in German locale. Could you miss it?